packages/material-ui/src/Autocomplete/Autocomplete.js

```diff
@@ -102,7 +102,7 @@ export const styles = (theme) => ({
         padding: '2.5px 4px',
       },
     },
-    '&.MuiFilledInput-root': {
+    '&[class*="MuiFilledInput-root"]': {
```

Comment:
Why were the `MuiFilledInput-*` changes reverted?

Reply:
Oops, I expected rebasing to fix this but forgot to check.